### PR TITLE
thelio-major-r2.2

### PR DESCRIPTION
**Pull request by jackpot51**
This sets the microphone device when the correct USB audio codec is found

### CODE REVIEW

File: system76driver/userdaemon.py [hunk 1]
```diff
@@ -139,6 +139,10 @@ def __init__(self, model, rootdir='/'):
         if self.model == "thelio-mira-b1":
             self.mic_dev = 3
             self.line_in_dev = 2
+        elif subprocess.call(["lsusb", "-d", "0414:a00d"]) != "":
```
Comment (jackpot51) [inline system76driver/userdaemon.py:142]: @leviport I think this does need to be 0 instead of ""

Comment (leviport) [reply system76driver/userdaemon.py:142]: Hmm, `lsusb -d 0414:a00d` was giving me a long string on the rev1.2 board and no output on other machines. Should I be expecting a `0` instead?

Comment (jackpot51) [reply system76driver/userdaemon.py:142]: Yes. Try running this in python3 interactive shell:

```
import subprocess
subprocess.call(["lsusb", "-d", "0414:a00d"])
```

You will see it return 0 on the rev 1.2 board and 1 on any other system.

Comment (leviport) [reply system76driver/userdaemon.py]: Yep, you're right. I misunderstood what `subprocess.call()` would do.

**Review by leviport — APPROVED** [review]
This is working great now!

**Pull request merged**